docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Ingesting Dynamo DB tables into the Landing Zone
+description: "Ingesting tables from a Dynamo DB instance into the Data Platform landing zone"
+layout: playbook_js
+tags: [playbook]
+---
+
+## Objective
+
+To setup a job that will ingest specified tables from a Dynamo DB instance.
+
+## Intended audience
+
+- Data Engineers
+- Data Analysts
+
+## Prerequisites
+
+- You have a Github account - you can [create one][github_signup] yourself using your Hackney email
+- You have been added to the 'LBHackney-IT' team - you can request this from Rashmi Shetty
+- You have an engineer in the account where the Dynamo DB instance is who can create an IAM role for you in that account.
+
+
+## Creating a role in the account where the Dynamo DB instance is
+
+You will need someone who has access to the account where the Dynamo DB tables live to help with this.
+They will need to create two roles, for both our pre-production and production AWS accounts.
+The roles will both need to have a trust policy that looks like this. One will hold the pre-production account number and one the production account number.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Principal": {
+                "AWS": "<account number of data platform account>"
+            }
+        }
+    ]
+}
+```
+Both roles should have the following policy attached to the role.
+Where ` -- TABLE ARNs HERE -- ` can be replaced by a list of ARNs for the tables that you want to ingest. 
+```
+{
+  "Version": "2012-10-17"
+  "Statement" [
+    {
+      "Effect" = "Allow"
+      "Action" = [
+        "dynamodb:List*",
+        "dynamodb:Get*",
+        "dynamodb:Query",
+        "dynamodb:Scan",
+        "dynamodb:BatchGet*",
+        "dynamodb:DescribeTable*",
+        "dynamodb:DescribeStream*",
+      ]
+      "Resource" = [
+        -- TABLE ARNs HERE --
+      ]
+    }
+  ]
+}
+```
+
+You will need the role ARN's of both roles, once they have been created, for the next step.
+
+## Saving the role ARN in the data platform accounts
+
+Complete this step in both pre-production & production AWS accounts.
+
+1. Login to AWS via [Hackney's SSO][hackney_sso].
+1. Navigate to [SSM parameter store][aws_console_ssm] in the AWS console.
+1. Click "Create parameter".
+1. Set the name of the parameter to be something like `/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-tables`.
```

Comment:
I would make it a bit more explicit that the respective environment name is used in the parameter name. Concerned someone might set the parameter name to have "prod" for both staging and production accounts

`/where-the-data-is-coming-from/name-of-environment/role-arn-to-access-dynamodb-tables`